buddy_config.py

```diff
@@ -62,7 +82,7 @@ def wrapper(self):
             except KeyError:
                 raise ConfigurationError(setting_func.var_name, setting_func.__name__)
             else:
-                if not res and setting_func.var_name not in self._defaults:
+                if res is None:
```

Comment:
At this point, it was already taken from from the defaults. So if it is None, then we have a situation.